src/sentry/tasks/embeddings_grouping/backfill_seer_grouping_records_for_project.py

```diff
@@ -89,11 +134,13 @@ def backfill_seer_grouping_records_for_project(
 
```

Comment:
yes, and a few reasons: 1) the row got expired in clickhouse, and for nodestore 2) it's not so much that there were no nodestore rows, but that the stacktraces _in_ nodestore weren't eligible for seer.

>do some of the ones we're backfilling already have embeddings (and if so, why are we backfilling them
yes, in case the backfill gets ran twice or something. we don't backfill them.

> What is the hashes dict, and what would make it empty?
it is the mapping of group hashes to group_id 

>Possibly out of scope for this PR, since it's not new here, but reading the backfill code for the first time, it's not clear quite what's happening here between lines 130 and 166. Why are we filtering the snuba results? has_snuba_row and has_nodestore_row imply that there might be some without snuba and/or nodestore rows? How/when would that happen? Similarly, with_no_embeddings... do some of the ones we're backfilling already have embeddings (and if so, why are we backfilling them)? What is the hashes dict, and what would make it empty?

> Could we add a few comments explaining what's happening and why?

if we end up needing to put a lot more work into the backfill, yes, but i imagine as we release this featue we'll end up deleting it soon, so could add comments if it becomes opaque to us as we work o it.